code/modules/griffening_cardgame/carddatums.dm

```diff
@@ -0,0 +1,685 @@
+	desc = "A Research Director must be in play and you own it to play this card. When this card is played, view your opponents hand and take one card."
+	Attributes = list(RESEARCH_DIRECTOR_REQUIRED, OPPONENT_HAND_SEARCH, DRAW_OPPONENT_HAND_CARD)
+
+/datum/griffeningcard/effect/deconstructor
+
+	name = "Deconstructor"
+	desc = "This card can only be played if facedown and used as a counter to a effect or equipment card and only if you control a scientist. Destroy the card, if this happens then search the opponents deck for a copy of the card. If a copy of the card has been found, discard that card and shuffle the deck."
+	Attributes = list(FACEDOWN_ONLY, SCIENTIST_REQUIRED, STOP_EQUIPPING, STOP_EFFECT, SEARCH_OPPONENT_DECK_FOR_COPY)
+
+/datum/griffeningcard/effect/enginesabotage
+
+	name = "Engine Sabotage"
+	desc = "Continuous effect, requires a antag to play, while this card is active, all opponent area cards are negated and no new area cards can be played. If there's a 'emergency shuttle' area card in play, the effects of it is not negated, but neither player can play a 'emergency shuttle' card."
+	Attributes = list(ANTAG_ONLY, CONTINUOUS, NULLIFY_OPPONENT_AREA
+
+/datum/griffeningcard/equipment/handcuffs
+
+	name = "Handcuffs"
+	desc = "Cannot be put facedown, target a human creature and equip this card to it. At the end of your third turn, the turn you play it on being the first, destroy the card. As long as that human has this card equipped, it cannot use it's effect or attack."
+	Attributes = list(NO_FACEDOWN, ANY_HUMAN, EQUIPMENT, TARGETTED, THREE_TURN_INCAPACITATE, THREE_TURN_DISCARD)
+
+/datum/griffeningcard/equipment/incendiarygrenade
+
+	name = "Incendiary Grenade"
+	desc = "If the creature this card is equipped to attacks, any opponent creature that has less than 20 DEF is destroyed. All other opponent creatures that have 20 or higher DEF will instead have DEF lowered by 20 until the end of your turn. If this effect has concluded, destroy this card."
+	REMOVEDEF = 20
+	Attributes = list(OPPONENT_REMOVEDEF_20_ONE_TURN, DISCARD_AFTER_EFFECT, EQUIPMENT, TARGETTED, DISCARD_BELOW_DEF)
+
+/datum/griffeningcard/effect/firefightinggrenade
+
+	name = "Fire Fighting Grenade"
+	desc = "This card may be played in response to Flamethrower, Plasma Fire and Incendiary Grenade. Instantly destroy both cards. When this card is activated, the opponent's turn immediately ends."
+	Attributes = list(END_OPPONENT_TURN, NULLIFY_FIRE, DISCARD_AFTER_EFFECT, ANY_TIME)
+
+/datum/griffeningcard/equipment/plasmafire
+
+	name = "Plasma Fire"
+	desc = "This card can only be played if there's a atmospheric tech in play. When this card is played, instantly reduce the DEF of all opponent humans by 10. While this card is active, all opponent humans lose 20 DEF at the start of the opponent's turn. If a creature reaches 0 DEF due to the effects of this card, the creature is killed. If any area cards are played while Plasma Fire is active, discard Plasma Fire."
```

Comment:
Continuous Effect. Requires Atmos Tech to play. -10 DEF to all opponent humans.  At the start of opponent's turn, -10 DEF on all opponent human cards (stacks). If they would have no defense, kill creature instead.

Card is discarded if an area card is played.